drivers/gpio/Kconfig.altera_nios2

```diff
@@ -0,0 +1,34 @@
+# Kconfig.altera_nios2 - Altera Nios-II GPIO configuration options
+#
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig GPIO_ALTERA_NIOS2
+	bool "Altera Nios-II PIO Controllers"
+	depends on GPIO && SOC_NIOS2F_ZEPHYR
```

Comment:
> SOC_NIOS2F_ZEPHYR

Huh? Get rid of this! This should not depend on a specific CPU build, that's just wrong...

Reply:
Ok will fix this.